src/Uno.UI/UI/Xaml/RoutedEventFlag.cs

```diff
@@ -83,12 +84,13 @@ internal static class RoutedEventFlagExtensions
 			  RoutedEventFlag.GotFocus
 			| RoutedEventFlag.LostFocus;
 
-		private const RoutedEventFlag _isDragAndDrop = (RoutedEventFlag)0b0000_0000_0000_0000___0000_0000_0001_1111___0000_0000_0000_0000___0000_0000_0000_0000;
-			//  RoutedEventFlag.DragEnter
-			//| RoutedEventFlag.DragLeave
-			//| RoutedEventFlag.DragOver
-			//| RoutedEventFlag.Drop
-			//| RoutedEventFlag.DropCompleted;
+		private const RoutedEventFlag _isDragAndDrop = // 0b0000_0000_0000_0000___0000_0000_0011_1111___0000_0000_0000_0000___0000_0000_0000_0000;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)